Taskfile.yaml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+tasks:
+  unit-test:
+    - go test -v ./...
+  unit-test-with-report:
+    - export GOPATH="$HOME/go/"
+    - export PATH=$PATH:$GOPATH/bin
+    - go install github.com/jstemmer/go-junit-report@latest
+    - go test -v ./... -coverpkg=./... -coverprofile cover.out | go-junit-report -set-exit-code > report.xml
+    - go tool cover -html cover.out -o coverage.html
```

Comment:
should we have line 10 and 11 only?

Reply:
yeah, we should, and remove go-junit-report from line 10